src/globals/scss/_layout.scss

```diff
@@ -4,12 +4,20 @@ $breakpoints: (
   bp--xs--major: 500px,
   bp--sm--major: 768px,
   bp--md--major: 992px,
-  bp--lg--major: 1200px
+  bp--lg--major: 1200px,
+);
+
+$exp-breakpoints: (
+  sm: 320px,
```

Comment:
Given change in variable name tends to a breaking change, 🤔 of a way not having to change this variable name before shipping the code in production. Wondering if one of the following would make sense:

1. (Given there no conflict in item names) Merging new breakpoints to `$breakpoints`, only in experimental mode
2. Figure out a name that we don't have to change before shipping the code in production

Reply:
That's a good point. Perhaps it would make the most sense to just add them to the existing map regardless.

Reply:
Checking with @joshblack but I believe the updated breakpoints should be pulled in from the layouts package in carbon-elements

Reply:
@tw15egan Thank you for sharing your thoughts! Just merging the two will make sense unless we see it's too early to incorporate `carbon-elements` (thank you for bringing this up @alisonjoseph!) or we see the breakpoints will change in future (meaning, the list of breakpoints is not seen as experimental). Using its [`carbon-elements`' `_breakpoint.scss`](https://github.com/IBM/carbon-elements/blob/v0.0.1-alpha.2/packages/layout/scss/_breakpoint.scss) is fine from my perspective, while we need to evaluate `carbon-elements` incorporation case-by-case basis given its nature of new fundamental codebase.

Reply:
For now, I've just added them to the existing breakpoint mixin. When we start using Carbon Elements, we will need to update all components

Reply:
Makes sense - Thanks @tw15egan for your quick update!